ci/default.nix

```
@@ -17,7 +17,13 @@ let
     else
       nixpkgs;
 
-  pkgs = import nixpkgs' { inherit system; };
+  pkgs = import nixpkgs' {
+    inherit system;
+    # Although it's not recommended, some contributors choose to disable aliases globally.
```

Maybe it's unnecessary to comment on whether or not it is recommended here?

```suggestion
    # Some contributors choose to disable aliases globally.
```

---

> I'm not sure whether I agree on the "not recommended" part. I think it's totally fine to do that, especially the way we use it.

I was intending to say that disabling it globally isn't recommended, not to say that disabling it on specific instances with known/pinned sources.

My justification is that disabling aliases globally is bound to cause issues when getting different and/or arbitrary revisions of nixpkgs. Or when using other people's configurations, flakes, etc.

But I agree. This comment isn't the place to say this, and it could easily be confused for saying what we're doing here isn't recommended.